doc/parameters.md

```diff
@@ -447,3 +447,27 @@ connections.
 
 By default Gateway does not enable it, and the keepalive timeout on nginx is set
 to [75 seconds](http://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_timeout)
+
+
+### `APICAST_CACHE_STATUS_CODES`
```

Comment:
I think we should clarify in these two envs that they're only used when the content caching policy is enabled.